huxley/www/static/js/huxley/components/AdvisorAssignmentsView.js

```diff
@@ -163,6 +187,35 @@ var AdvisorAssignmentsView = React.createClass({
     }
   },
 
+  _handleFinalizeNames: function(event) {
+    /* Would appreciate some suggestions here. There are two things we need to do:
+      1. Update each delegate to have its assignment field set here.
```

Comment:
I think django rest framework has a concept of bulk editing. I imagine we'd send a bulk PATCH request to the delegate API and set all of their assignment_id fields to the relevant assignment.